Podfile

```diff
@@ -1,12 +1,11 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '9.0'
+# platform :ios, '9.0'
```

Comment:
I think such an old Swift version is overkill and results in older framework versions.